en-US/Conductor_Handbook/themes/pdf-theme.yml

```diff
@@ -0,0 +1,42 @@
+extends:
+  - default
+page:
+  size: Letter
+title-page:
```

Comment:
With these settings, the title page for the Conductor Handbook currently looks like this:
<img width="728" height="947" alt="image" src="https://github.com/user-attachments/assets/71e80dda-1333-41f8-8bb2-da0b46c6fef8" />